websocket/README.md

```diff
@@ -0,0 +1,77 @@
+## Wing WebSocket Support
+
+This library allows you tu create WebSocket usiing Wing.
```

Comment:
Add a little background and some links about websockets and why would someone wants to use it